deploy/Chart/templates/database/configmaps.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
```

Comment:
Let's sync on this tomorrow, this is a really good start, and I think we should generate a list of TODO follow ups so we can decide what's in scope for this PR and what's not 👍